update README.md (#65)
mitodl/mit-open: README.md

    Sets the hostname required by webpack for building the frontend. Should likely be whatever you set 
    the host to in your /etc/hosts or the hostname that you're accessing it from. Likely `od.odl.local`.

# Code Generation
MIT Open uses [drf-spectacular](https://drf-spectacular.readthedocs.io/en/latest/) to generate and OpenAPI spec from Django views. Additionally, we use [OpenAPITools/openapi-generator](https://github.com/OpenAPITools/openapi-generator) to generate Typescript declarations and an API Client. These generated files are checked into source control; CI checks that they are up-to-date. To regenerate these files, run
```bash
./scripts/generate_openapi.sh
```

# Committing & Formatting

> To ensure commits to GitHub are safe, you should install the following first:

Reviewer: ```suggestion
To ensure commits to GitHub are safe, first install [pre-commit](https://pre-commit.com/):
```
